haystack/backends/solr_backend.py

```diff
@@ -48,6 +55,14 @@ def __init__(self, connection_alias, **connection_options):
         self.conn = Solr(connection_options['URL'], timeout=self.timeout, **connection_options.get('KWARGS', {}))
         self.log = logging.getLogger('haystack')
 
+    def get_schema_admin(self):
```

Comment:
At first I was thinking that we should just use `self.schema_admin` rather than underscore-prefixing it, but then I wanted to get your opinion on using Django's [`cached_property`](https://docs.djangoproject.com/en/1.9/ref/utils/#django.utils.functional.cached_property) on this instead.